Refactor Registries (#1889)
spectrocloud/librarium: docs/docs-content/registries-and-packs/registries/registries.md

## Synchronization

Registries are automatically synchronized with Palette. Helm registries are different as you can choose to disable synchronization. When you disable synchronization, you must manually synchronize the Helm registry with Palette. Refer to the [Helm Registry](helm-charts.md#synchronization-behavior) resource to learn more about the synchronization behavior of Helm registries.

## Default Registries

Palette comes with a set of default registries that are available to all SaaS tenants, and non-airgap self-hosted Palette environments. The default registries are listed below:

| **Name** | **Provider** | **Description** | **URL** | **Base Path** |
| --- | --- | --- | --- | --- | --- |
| Bitnami | Helm | A Helm Chart registry containing Helm Charts maintained and supported by Bitnami. | `https://charts.bitnami.com/bitnami` | - |

> | Public Spectro Helm Repo | Helm | A Helm Chart registry containing Helm Charts maintained and supported by us. | `https://spectrocloud.github.io/helm-charts` | - |

Reviewer: ```suggestion
| Public Spectro Helm Repo | Helm | A Helm Chart registry containing Helm Charts maintained and supported by Spectro Cloud. | `https://spectrocloud.github.io/helm-charts` | - |
```
Although we are consistent in using "we" to refer to Spectro Cloud, it still felt a little jarring for me to read "supported by us". I think it's better to be specific

> | Public Repo | Legacy Packs | A packs registry containing packs maintained and supported by us. |  `https://registry.spectrocloud.com` | - |

Reviewer: ```suggestion
| Public Repo | Legacy Packs | A packs registry containing packs maintained and supported by Spectro Cloud. |  `https://registry.spectrocloud.com` | - |
```

| Spectro Addon Repo | Legacy Packs | A packs registry containing add-on packs maintained and supported by us. | `https://registry-addon.spectrocloud.com` | - |

> | Palette Registry | OCI | A packs registry containing packs maintained and supported by us. | `415789037893.dkr.ecr.us-east-1.amazonaws.com` | `production` |

Reviewer: ```suggestion
| Palette Registry | OCI | A packs registry containing packs maintained and supported by Spectro Cloud. | `415789037893.dkr.ecr.us-east-1.amazonaws.com` | `production` |
```

| Palette Community Registry| OCI | A packs registry containing community packs. | `415789037893.dkr.ecr.us-east-1.amazonaws.com` | `community` |
 
:::info

> Palette VerteX comes with a default OCI registry that only contains FIPS compliant packs. Non-FIPS compliant packs are not available in Palette VerteX by default and must explicitly be added to Palette VerteX. Refer to the [Use non-FIPS Packs](../../vertex/system-management/enable-non-fips-settings/enable-non-fips-settings.md) guide to learn how to add non-FIPS packs registries to Palette VerteX.

Reviewer: ```suggestion
Palette VerteX comes with a default OCI registry that only contains FIPS compliant packs. Non-FIPS compliant packs are not available in Palette VerteX by default and must explicitly be added to Palette VerteX. Refer to the [Use non-FIPS Packs](../../vertex/system-management/enable-non-fips-settings/enable-non-fips-settings.md) guide to learn how to add non-FIPS packs to Palette VerteX.
```
